data/remnant ships.txt

```diff
@@ -209,8 +209,7 @@ ship "Gascraft"
 		"Emergency Ramscoop"
 		"Quantum Key Stone"
 
-		"Crucible-Class Thruster"
-		"Crucible-Class Steering"
+		"Cassette-Class Engine"
```

Comment:
You forgot to rename the outfit here and in the outfitter.

Reply:
Thanks! Fixed.